VocabularyTrainer/Home Screen/HomeLanguageHeaderView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  HomeLanguageHeaderView.swift
+//  VocabularyTrainer
+//
+//  Created by Michael Steudter on 19.12.22.
+//  Copyright © 2022 mic. All rights reserved.
+//
+
+import UIKit
+
+protocol HomeLanguageHeaderViewDelegate: AnyObject {
+    func tappedPracticeButton()
+    func tappedEditButton()
+    func tappedAddLanguageButton()
+}
+
+final class HomeLanguageHeaderView: UIView {
+
+    typealias Strings = HomeViewModel.Strings
+    typealias Colors = HomeViewModel.Colors
+
+    /// Title with trailing add button.
+    private let titleLabel = UILabel()
+    /// Button for adding a new language.
+    lazy var addLanguageButton: UIButton = {
+        let button = UIButton(type: .contactAdd)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.tintColor = .label
+        button.addAction(.init(handler: { [weak self] _ in self?.delegate?.tappedAddLanguageButton() }), for: .touchUpInside)
```

Comment:
Suggestion: break line - swiftlint stuff:
```
button.addAction(.init(handler: { [weak self] _ in
    self?.delegate?.tappedAddLanguageButton()}),
                 for: .touchUpInside)

```

Reply:
I agree, makes sense. It's currently a little hard to read in one line

Reply:
updated here: 3562f2d8fdb5f668d44473b564f64b5f216d0bac